pdns/dnsdist-lua.cc

```diff
@@ -1967,13 +1967,7 @@ vector<std::function<void(void)>> setupLua(bool client, const std::string& confi
   setupLuaRules();
   setupLuaVars();
 
-  std::ifstream ifs(config);
-  if(!ifs)
-    warnlog("Unable to read configuration from '%s'", config);
-  else
-    vinfolog("Read configuration from '%s'", config);
```

Comment:
Perhaps we could keep this line, after the `g_lua.executeCode()` one since we don't need to have the name of the file printed before anymore.

Reply:
yes, makes sense